Mathlib/SetTheory/Ordinal/Topology.lean

```diff
@@ -180,15 +183,18 @@ theorem isLimit_of_mem_frontier (ha : a ∈ frontier s) : IsLimit a := by
   subst hb; subst hc
   exact hc' hb'
 
-theorem isNormal_iff_strictMono_and_continuous (f : Ordinal.{u} → Ordinal.{u}) :
+@[deprecated (since := "2025-07-08")]
+alias isLimit_of_mem_frontier := isSuccLimit_of_mem_frontier
+
+theorem isNormal_iff_strictMono_and_continuous (f : Ordinal.{u} → Ordinal.{max u v}) :
```

Comment:
What happened here? This is another change than the titular one. Can it be in another PR?

Reply:
I think a partial refactor of this file might have snuck through, let me revert that.